src/Dapr.Client/InvocationHandler.cs

```diff
@@ -98,63 +96,68 @@ internal string? DaprApiToken
         /// <inheritdoc />
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            var original = request.RequestUri;
-            if (!this.TryRewriteUri(request.RequestUri, out var rewritten))
+            var originalUri = request.RequestUri;
+            
+            if (!this.TryRewriteUri(request.RequestUri, out var rewrittenUri))
             {
-                throw new ArgumentException($"The request URI '{original}' is not a valid Dapr service invocation destination.", nameof(request));
+                throw new ArgumentException($"The request URI '{originalUri}' is not a valid Dapr service invocation destination.", nameof(request));
             }
 
             try
             {
                 var apiTokenHeader = DaprClient.GetDaprApiTokenHeader(this.apiToken);
+                
                 if (apiTokenHeader is not null)
                 {
                     request.Headers.Add(apiTokenHeader.Value.Key, apiTokenHeader.Value.Value);
                 }
-                request.RequestUri = rewritten;
+                
+                request.RequestUri = rewrittenUri;
 
                 return await base.SendAsync(request, cancellationToken);
             }
             finally
             {
-                request.RequestUri = original;
+                request.RequestUri = originalUri;
                 request.Headers.Remove("dapr-api-token");
             }
         }
 
         // Internal for testing
-        internal bool TryRewriteUri(Uri? uri, [NotNullWhen(true)] out Uri? rewritten)
+        internal bool TryRewriteUri(Uri? uri, [NotNullWhen(true)] out Uri? rewrittenUri)
```

Comment:
Same thing - I'm disinclined to accept this change that renames the argument just because you can. Again, if you're changing the method itself and need to change the name so it's useful in your rewrite, that's one thing, but changing it just for the sake of doing so here, especially for something that's strictly internal for testing purposes, doesn't make sense.